src/diffusers/loaders.py

```diff
@@ -2598,7 +2649,6 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
 
         is_model_cpu_offload = False
         is_sequential_cpu_offload = False
-        recursive = False
```

Comment:
No need to introduce a new concept if only used in one function call with `recursive=...`